Sources/App/Models/License.swift

```diff
@@ -151,6 +151,15 @@ enum License: String, Codable, Equatable, CaseIterable {
         case other
         case incompatibleWithAppStore = "incompatible"
         case compatibleWithAppStore = "compatible"
+        
+        var userFacingString: String {
+            switch self {
+            case .none: return License.none.shortName
+            case .other: return License.other.shortName
+            case .incompatibleWithAppStore: return "App Store incompatible"
```

Comment:
This wording seems off. My first instinct was to have this say "incompatible with the App Store" but when it's in the filter it says "matches" and so:

"matches incompatible with the App Store" is weird

but so is:

"matches App Store incompatible"

It makes me think that "matches" might be better as "is". I think "is" would work everywhere "matches" works and would make this make more sense too.